lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +22,96 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  color: $nx-text-color-light;
```

Comment:
Either this is redundant or the `color` on `.nx-list` is

Reply:
> Either this is redundant or the `color` on `.nx-list` is

They all should be with the work you did standardizing the colours so I have removed them here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/a4627bcaa98ac3cd0036f79784f12b5a77416963